redis/commands/core.py

```diff
@@ -12,14 +12,7 @@
 )
 
 
-class CoreCommands:
-    """
-    A class containing all of the implemented redis commands. This class is
-    to be used as a mixin.
-    """
-
-    # SERVER INFORMATION
-
+class AclCommands:
```

Comment:
Ongoing nit for these classes. If we're going to separate commands by class (which I'm in favour of), then the classes need docstrings.

Reply:
Also - while you're here, if we're going to split up the classes, rather than have ~3.5K worth of classes in a single file, let's split the files up (i.e redis.commands.acl contains ACLCommands.

Reply:
In cases where something is an acryonym (i.e ACL) we should use ACLCommands. In cases (i.e pubusb) where it's a word, we should stick to what you've done and use PubSubCommands.